homeassistant/components/lg_soundbar/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "LG Soundbar Connection details",
+        "description": "Enter your Connection details.",
```

Comment:
These don't add any value. The title is automatically injected from the manifest.
```suggestion
```